tests/core/pyspec/eth2spec/test/helpers/genesis.py

```diff
@@ -176,6 +176,8 @@ def create_genesis_state(spec, validator_balances, activation_threshold):
 
     if is_post_electra(spec):
         state.deposit_requests_start_index = spec.UNSET_DEPOSIT_REQUESTS_START_INDEX
+        # Initialize proposer lookahead list
+        state.proposer_lookahead = spec.compute_proposer_lookahead(state)
```

Comment:
We'll need to move this into a new `is_post_fulu(spec)` block.

Reply:
Good find! Fixed in https://github.com/ethereum/consensus-specs/pull/4190/commits/6a1d923730c0b368b5192e64f643f22c21451058